homeassistant/components/ezviz/update.py

```diff
@@ -73,11 +73,9 @@ def installed_version(self) -> str | None:
         return self.data["version"]
 
     @property
-    def in_progress(self) -> bool | int | None:
+    def in_progress(self) -> bool:
         """Update installation progress."""
-        if self.data["upgrade_in_progress"]:
-            return self.data["upgrade_percent"]
-        return False
+        return bool(self.data["upgrade_in_progress"])
```

Comment:
Maybe the bool conversion is not needed: https://github.com/BaQs/pyEzviz/blob/master/README.md?plain=1#L134
@RenierM26 can you confirm?